scripts/lib/setup/installers.sh

```diff
@@ -216,20 +216,26 @@ function install_node_via_nvm() {
   cd "$(repo_path)"
 
   if [ ! -e "$nvmrc" ]; then
-    cecho "@b@blue[[+ Installing Node 9 (Node 10 is not supported by Realm)]]"
+    cecho "@b@blue[[+ Installing Node 8 (Node 10 is not supported by Realm and Node 9 doesn't support npm 5.5.1)]]"
 
-    nvm install 9
-    nvm alias status-im 9
+    nvm install 8.9.4
+    nvm alias status-im 8.9.4
     echo status-im > "$nvmrc"
 
     nvm use status-im
+    npm install -g npm@5.5.1
```

Comment:
nvm should install npm too, the correct revision for the specific node version. Why override?

Reply:
I remember there used to be problems with 5.6.x which node installed, and the documentation asked to downgrade to 5.5.1. Again, this is based on instructions from `Build Desktop` page.

Reply:
In the end I don't mind removing this from the script, if we can update/remove the requirements from the documentation. I just don't like the duality of having the documention requesting something and then our scripts installing another.

Reply:
at least, it is worth a comment in the script :)

Reply:
I agree that strictly sticking to `5.5.1` is the way to avoid madness. I've wasted far too much time debugging issues just because the npm version was off a bit.
We really need yarn...

Reply:
Good point about the comment. Added it, and also added an additional commit to the PR to help diagnose when Metro can't start due to port 8081 being taken. Something like this will be shown in that case:

![image](https://user-images.githubusercontent.com/138074/48935186-086d7200-ef07-11e8-8054-329701da9587.png)